### ISSUE TITLE
Unable to scroll pages with touchpad, and stylus GTK input issues

### ISSUE THREAD

**Issue by steewbsd** [xournalpp/xournalpp#4319]
(Please complete the following information, and then delete this line)

**Affects versions :**
 - OS: Linux 6.0.0 Alpine edge (affects previous versions too)
 - (Linux only) Desktop environment: GNOME-Wayland
 - Which version of libgtk do you use
     -  3.24.34
 - Version of Xournal++: [e.g. 1.0.16, specific commit, specific branch]
     - 1.1.1
 - Installation method: Alpine's apk manager
**Describe the bug**
I seem to be unable to scroll any page, let it be PDF pages, or blank pages, using the laptop touchpad. I can scroll them with the hand tool.
Additional to this, the stylus (Huion H610 Pro) is not able to work as an input system in the color chooser, preferences menu and some other GTK menus. It does work in the main toolbars, dropdowns and of course, drawing.

**To Reproduce**
Open program normally, try to scroll with touchpad results in no behavior whatsoever.
Open any GTK menu, like preferences or the color chooser, and try to click buttons with the stylus, no response either.

**Screenshots of Problem**
![image](https://user-images.githubusercontent.com/26441684/195797549-a1e97cbe-0094-455a-9c93-cda32c4bb811.png)

**Comment by rolandlo** [comment]
> Additional info: seems like the flatpak version works correctly. Issue only happens in alpine's packaged version.

Does that addition info concern both issues (touchpad scrolling and stylus)?

For scrolling do you use 2 fingers?

**Comment by steewbsd** [comment]
> > Additional info: seems like the flatpak version works correctly. Issue only happens in alpine's packaged version.
> 
> Does that addition info concern both issues (touchpad scrolling and stylus)?
> 
> For scrolling do you use 2 fingers?

Yes, it fixes both issues (though it still has some quirks regarding stylus and GTK dialogs, but at least color picker works now), and I scroll with two fingers on the touchpad.
